eng/common/pipelines/templates/steps/replace-relative-links.yml

```diff
@@ -0,0 +1,217 @@
+parameters: 
+  TargetFolder: ''
+  RootFolder: ''
+  BuildSHA: ''
+  RepoId: ''
+
+steps:
```

Comment:
I know @KarishmaGhiya was planning to replace this template so we should figure out if we should wait until then to update usages of this template.